src/main/java/uk/gov/gchq/palisade/service/request/Policy.java

```diff
@@ -328,13 +287,13 @@ public boolean equals(final Object o) {
         if (this == o) {
             return true;
         }
-        if (!(o instanceof Policy)) {
+        if (o == null || getClass() != o.getClass()) {
```

Comment:
Update equals method to use the new equals method which uses instanceOf

Reply:
Done as part of a re-do of all of the equals methods